frontend/src/lib/hooks/useAnchor.ts

```diff
@@ -11,11 +11,8 @@ export function useAnchor(hash: string): void {
             }
 
             element.classList.add('highlighted')
-            const elementPosition = element.getBoundingClientRect().top + window.pageYOffset
-            window.scrollTo({
-                top: elementPosition - 50 - 32, // compensate for header & top margin of pages
-                behavior: 'smooth',
-            })
+
```

Comment:
noticed this only works for me the second time I click the link

(or rather not the first time after a refresh)

![2023-08-17 17 45 10](https://github.com/PostHog/posthog/assets/984817/99e56867-0c1d-4a1e-af9c-1d79e38cf4e0)

Reply:
Fixed with a `window.requestAnimationFrame`